dhall-docs/src/Dhall/Docs.hs

```diff
@@ -51,20 +68,116 @@ parseOptions :: Parser Options
 parseOptions =
         Options
     <$> Options.Applicative.strOption
-        ( Options.Applicative.long "path"
-       <> Options.Applicative.metavar "PATH"
-       <> Options.Applicative.help "Root folder of your dhall package" )
+        ( Options.Applicative.long "input"
+       <> Options.Applicative.metavar "INPUT"
+       <> Options.Applicative.help "Directory of your dhall package. It should \
+                                    \contain only text files" )
```

Comment:
Is the text-only restriction necessary?  We could ignore files that don't parse as valid Dhall code

Reply:
Well I left a TODO comment here:
https://github.com/dhall-lang/dhall-haskell/blob/470f9e359ec7443db7f1ff2955c7109d0655706d/dhall-docs/src/Dhall/Docs.hs#L92-L100

do you know of an util that checks if the file is binary? That gave me an error

Reply:
@german1608: You can read the file as a `ByteString` using `Data.ByteString.readFile` and then use `Data.Text.decodeUtf8'`.  Then if decoding fails you know the file is binary